packages/cashier-v2/src/lib/DepositFiat/DepositFiat.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+import { useAuthorize, useCashierFiatAddress } from '@deriv/api';
+import { Loader } from '@deriv-com/ui';
+import { ErrorScreen } from '../../components/ErrorScreen';
+import { isServerError } from '../../utils';
+import styles from './DepositFiat.module.scss';
+
+const DepositFiat = () => {
+    const { isSuccess: isAuthorizeSuccess } = useAuthorize();
+    const { data: iframeUrl, error: depositError, isError, mutate } = useCashierFiatAddress();
+    const [isLoading, setIsLoading] = useState(true);
```

Comment:
Here we are using custom `isLoading` state for a smoother transition from `Loader` to `IFrame`.